markdown/chapters/univariate-analysis/main.rmd

```diff
@@ -0,0 +1,1672 @@
+  }
+  if(ver6$all_2[i]=="A"){
+    ver6$all_2[i] <- "Wider"
+  }
+  
+  if(ver6$all_3[i]=="C"){
+    ver6$all_3[i] <- "Default"
+  }
+  if(ver6$all_3[i]=="B"){
+    ver6$all_3[i] <- "Narrower"
+  }
+  if(ver6$all_3[i]=="A"){
+    ver6$all_3[i] <- "Wider"
+  }
+}
+
+comp_ratio <- rbind(ver1[,c(4:7, 31:33)], ver2[,c(4:7, 31:33)], ver3[,c(4:7, 31:33)], ver4[,c(4:7, 31:33)], ver5[,c(4:7, 31:33)], ver6[,c(4:7, 31:33)])
+
+
+### STACKED
+vir_stacked <- rbind(ver1[,c(4:7, 34:41)], ver5[,c(4:7, 34:41)])
+def_stacked <- rbind(ver2[,c(4:7, 34:41)], ver3[,c(4:7, 34:41)])
+gr_stacked <- rbind(ver4[,c(4:7, 34:41)], ver6[,c(4:7, 34:41)])
+
+set_a <- ver1[,c(4:9, 42:44)]
+set_b <- ver2[,c(4:9, 42:44)]
+set_c <- ver3[,c(4:9, 42:44)]
+set_d <- ver4[,c(4:9, 42:44)]
+set_e <- ver5[,c(4:9, 42:44)]
+set_f <- ver6[,c(4:9, 42:44)]
+
+### SALES_AB
+ab_sep <- rbind(ver1[,c(4:7, 45:47)], ver2[,c(4:7, 45:47)], ver3[,c(4:7, 48:50)], ver4[,c(4:7, 51:53)], ver5[,c(4:7, 48:50)], ver6[,c(4:7, 51:53)])
+
+ab_trn <- rbind(ver1[,c(4:7, 48:50)], ver2[,c(4:7, 51:53)], ver3[,c(4:7, 45:47)], ver4[,c(4:7, 45:47)], ver5[,c(4:7, 51:53)], ver6[,c(4:7, 48:50)])
+
+ab_zero <- rbind(ver1[,c(4:7, 51:53)], ver2[,c(4:7, 48:50)], ver3[,c(4:7, 51:53)], ver4[,c(4:7, 48:50)], ver5[,c(4:7, 45:47)], ver6[,c(4:7, 45:47)])
+
+
+### SALES_CD
+cd_trn <- rbind(ver1[,c(4:7, 54)], ver2[,c(4:7, 54)], ver3[,c(4:7, 55)], ver4[,c(4:7, 54)], ver5[,c(4:7, 55)], ver6[,c(4:7, 55)])
+
+cd_zro <- rbind(ver1[,c(4:7, 55)], ver2[,c(4:7, 55)], ver3[,c(4:7, 54)], ver4[,c(4:7, 55)], ver5[,c(4:7, 54)], ver6[,c(4:7, 54)])
+
+
+
+```
+
+This chapter will discuss a basic univariate analysis of the survey results, in which we look at the summary statistics and make by-eye comparisons between groups withing the whole population.
+
+In the chapter 3 we will delve deeper into the data, sub-setting for various groups and performing comparisons between and within these groups to ascertain whether different demographic and population factors have an effect on responses.
+
+\subsection{Ninja Warrior - Part 1}
+
+The first part of the survey consisted of showing the respondents three differently scaled bar plots representing how many times four obstacles were used throughout 10 seasons of the television show American Ninja Warrior. The three presented visualisations all showed the same raw data, but each was produced with a different y-axis scaling, in order to assess whether changing the scale in these ways affects viewer interpretation of interpreting both differences and exact bar heights. Each respondent was asked four questions; two free form answer and two multiple choice. 
+
+The use of free form answers did result in occasional non-valid answers, such as statements along the lines of "Don't know" when a number was required. Most people also opted to write a number between 0 and 1 for the fourth question when a percentage was required. 
+
+**Approximately many times would you say the 'Salmon Ladder' was used?**
```

Comment:
This need something like:

```
The specific question for this is:
```

also (minor):

```suggestion
> **Approximately many times would you say the 'Salmon Ladder' was used?**
```

That will put the above in a block quote and might stand out "nicer" but have a look and decide if you like it like that.